nerdlets/groundskeeper-nerdlet/index.js

```diff
@@ -675,21 +671,16 @@ export default class Groundskeeper extends React.Component {
     const upToDateLabel = agentSloOptions[agentSLO].label;
     const scanner = scanIsRunning ? <Spinner inline /> : undefined;
 
-    const tableBannerText = slaReportKey
-      ? `SLA Report by ${slaReportKey}`
-      : tableState === 'outOfDate'
-      ? `${presentationData.outdatedTable.data.length} apps are running
-              outdated agents`
-      : tableState === 'multipleVersions'
-      ? `${presentationData.multiversionTable.data.length} apps are running
-              multiple agent versions`
-      : tableState === 'upToDate'
-      ? `${presentationData.currentTable.data.length} apps are up to date
-              with ${upToDateLabel}`
-      : tableState === 'noVersionReported'
-      ? `${presentationData.noVersionsTable.data.length} apps are not
-              reporting agent version data (they may be inactive)`
-      : undefined;
+    let tableBannerText = slaReportKey ? `SLA Report by ${slaReportKey}` : '';
```

Comment:
It's a bit weird to mix a ternary w/ an if/else block, and is tableState guaranteed to not match one of the if clauses below when slaReportKey is non-null? 
Seems better to assign `tableBannerText = ''` and  have the if/else block start w/ `if (slaReportKey) {...} else if (tableState === 'outOfDate') {...`

Reply:
Great call @fightingmonk. Making that change now.